pages/sponsors.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import Image from "next/image";
+import styles from "@/styles/Sponsors.module.scss";
+
+const laCtfLogos = [
+  { src: "/images/crowdstrike.png", alt: "CrowdStrike" },
+  { src: "/images/google_cloud.png", alt: "Google Cloud" },
+  { src: "/images/latticework.png", alt: "Latticework" },
```

Comment:
Also, can we make it so that logos can have links so we can link to sponsors' websites?